sign/ed25519/ed25519.go

```diff
@@ -22,117 +32,139 @@ type PrivateKey []byte
 type KeyPair struct{ private, public [Size]byte }
 
 // GetPrivate returns a copy of the private key.
-func (k *KeyPair) GetPrivate() PrivateKey { return makeCopy(&k.private) }
+func (k *KeyPair) GetPrivate() PrivateKey { z := k.private; return z[:] }
+
+// GetPublic returns a copy of the public key.
+func (k *KeyPair) GetPublic() PublicKey { z := k.public; return z[:] }
 
-// GetPublic returns the public key corresponding to the private key.
-func (k *KeyPair) GetPublic() PublicKey { return makeCopy(&k.public) }
+// Seed returns the private key seed.
+func (k *KeyPair) Seed() []byte { return k.GetPrivate() }
```

Comment:
`Seed` is the terminology used by Go's stdlib ed25519, because they view the SHA512 expansion as the private key, whereas the RFC and Dan define the "seed" as the private key.  I think `Private()` is more appropriate here.

Reply:
I share this though too, however the reason for providing these methods and these terminology is to serve as a drop-in replacement wrt stdlib (which doesn't use the best terms/interfaces)
Do you see value on providing these methods?

Reply:
Ah, being a drop-in replacement for the sodlib one is a good argument.  It's value is in interoperability.